<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../chameleon_logo.gif</normaloff>../chameleon_logo.gif</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>761</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="tittle">
     <item>
      <widget class="QLabel" name="logo">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../waiter.gif</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>URW Chancery L</family>
         <pointsize>72</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Garçon</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>781</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainMenu">
     <item>
      <layout class="QVBoxLayout" name="entres">
       <item>
        <widget class="QLabel" name="tittle_2">
         <property name="font">
          <font>
           <family>TeXGyreSchola</family>
           <pointsize>28</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>Entrés</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="list"/>
       </item>
       <item>
        <widget class="QPushButton" name="add">
         <property name="text">
          <string>Add One</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>=============================================================================================</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="Details">
       <item row="1" column="1">
        <widget class="QTextEdit" name="Year_Made"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="details">
         <property name="text">
          <string>Details : </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="year">
         <property name="text">
          <string>Year : </string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="chef">
         <property name="text">
          <string>Chef : </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTextEdit" name="Author_name"/>
       </item>
       <item row="2" column="1">
        <widget class="QTextEdit" name="Descriptions"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Bash">
         <property name="text">
          <string>Recipe : </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextEdit" name="BashLoc"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="actions">
       <item>
        <widget class="QPushButton" name="edit">
         <property name="text">
          <string>Edit Info</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="run">
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Remove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="QuitButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>700</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGARCON">
    <property name="title">
     <string>GARCON</string>
    </property>
   </widget>
   <addaction name="menuGARCON"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
